etc/config/c++.amazon.properties

```diff
@@ -1,4 +1,4 @@
-compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&clang-rocm:&rvclang:&wasmclang:&cl:&cross:&ellcc:&zapcc:&djggp:www.godbolt.ms@443:&armclang32:&armclang64:&zigcxx:&cxx6502
+compilers=&gcc86:&icc:&icx:&clang:&clangx86trunk:&clang-rocm:&rvclang:&wasmclang:&cl:&cross:&ellcc:&zapcc:&djggp:www.godbolt.ms@443:&armclang32:&armclang64:&zigcxx:&cxx6502:&nvcxx_x86_cxx:&nvcxx_arm_cxx
```

Comment:
The ARM compilers are failing to install right now; but I'll leave them in here for now as once they're working they'll appear after the next release.